packages/governance/test/swingsetTests/contractGovernor/governedContract.js

```diff
@@ -1,47 +1,43 @@
 // @ts-check
 
 import { Far } from '@agoric/marshal';
-import { E } from '@agoric/eventual-send';
 import { assert, details as X } from '@agoric/assert';
 import { sameStructure } from '@agoric/same-structure';
 
 import { buildParamManager, ParamType } from '../../../src/paramManager';
 
 const MALLEABLE_NUMBER = 'MalleableNumber';
 
-export const governedParameterTerms = {
-  contractParams: [MALLEABLE_NUMBER],
-};
+const governedParameterTerms = harden([MALLEABLE_NUMBER]);
 
-/* @type {ContractStartFn} */
+/** @type {ContractStartFn} */
 const start = async zcf => {
-  const { electionManager, governedParams } = zcf.getTerms();
-  const governorPublic = E(zcf.getZoeService()).getPublicFacet(electionManager);
+  /** @type {{ governor: Instance, governedParams: Array<string> }} */
+  const { governor, governedParams } = zcf.getTerms();
+
+  assert(
+    sameStructure(governedParams, governedParameterTerms),
+    X`governedParams must be ${governedParameterTerms}`,
+  );
+
   const paramManager = buildParamManager([
     {
       name: MALLEABLE_NUMBER,
-      value: 602214090000000000000000n,
+      value: 602214090000000000000000n, // initial value
       type: ParamType.NAT,
     },
   ]);
-  assert(
-    sameStructure(governedParams, harden(governedParameterTerms)),
-    X`Terms must include ${MALLEABLE_NUMBER}`,
-  );
-
-  const gCreator = await E(governorPublic).governContract(
```

Comment:
The contract developer wouldn't have to know anything about governance, just that they should define which parameters are governable and expose the paramManager and the governor instance.